rmw_fastrtps_cpp/include/rmw_fastrtps_cpp/custom_client_info.hpp

```diff
@@ -43,7 +45,7 @@ typedef struct CustomClientInfo
 typedef struct CustomClientResponse
 {
   eprosima::fastrtps::rtps::SampleIdentity sample_identity_;
-  eprosima::fastcdr::FastBuffer * buffer_;
+  std::unique_ptr<eprosima::fastcdr::FastBuffer> buffer_;
```

Comment:
why does this have to be a heap allocation in the first place?

Reply:
eProsima/Fast-CDR#19